python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py

```diff
@@ -696,7 +696,35 @@ async def latiss_take_sequence(self, silent=True):
             await self.atcs.rem.ataos.cmd_offset.set_start(z=-self.manual_focus_offset)
             self.manual_focus_offset_applied = False
 
+    async def assert_feasibility(self) -> None:
+        """Verify that the telescope in a feasible state to execute the
+        script.
+        """
+
+        await self.atcs.assert_all_enabled()
+        await self.latiss.assert_all_enabled()
+
+        self.log.debug("Check ATAOS corrections are enabled.")
+        ataos_corrections = await self.atcs.rem.ataos.evt_correctionEnabled.aget(
+            timeout=self.atcs.fast_timeout
+        )
+
+        assert (
+            ataos_corrections.hexapod
+            and ataos_corrections.m1
+            and ataos_corrections.atspectrograph
+        ), (
+            "Required ATAOS corrections not enabled: ",
```

Comment:
I think this might be a bit ambiguous. Consider:
`Not all required ATAOS corrections are enabled. The following loops must all be closed (True), but are currently:`